ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -9607,6 +9610,61 @@ void HqlCppTranslator::doBuildExprFailCode(BuildCtx & ctx, IHqlExpression * expr
     }
 }
 
+void HqlCppTranslator::doBuildExprMatchedInDenormalizeGroup(BuildCtx & ctx, HqlExprAssociation *rowCountMarker, IHqlExpression * expr, CHqlBoundExpr & tgt)
+{
+    IHqlExpression * matchExpr = expr->queryChild(0);
+    assertex(matchExpr);
+
+    node_operator op = matchExpr->getOperator();
+    if (op==no_left)  //included for completeness, always return true
+    {
+        tgt.expr.setown(createConstant(createBoolValue(true)));
```

Comment:
@ghalliday I don't think I can make this assumption so may have to pass flags for matching left here as well.  But please go ahead and review the rest.